src/test/java/tests/RestTests/UserTests.java

```diff
@@ -0,0 +1,62 @@
+package tests.RestTests;
+
+
+import com.endava.models.CreatePersonRequest;
+import io.restassured.response.Response;
+import restActions.BaseRestActions;
+import restActions.UserRestActions;
+import com.endava.models.CreatePersonResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class UserTests {
+
+    private  UserRestActions userRestActions;
+
+
+    @BeforeEach
+    public void setup() {
+        userRestActions = new UserRestActions();
+    }
+
+    @Test
+    public void getSingleUser() {
+     userRestActions.getResource("1", 200);
+    }
+
+    @Test
+    public void verifyUnexistingUserRequest() {
+        userRestActions.getUnexistingUser();
+    }
+
+    @Test
+    public void verifyListOfUsers() {
+        userRestActions.getListOfUsers();
+    }
+
+    @Test
+    public void verifyFirstObjectMatchesExpected() {
```

Comment:
Refactor underlying methods and return response to the test class; then include assertions about the response.
firstObjectValidation does not convey what is to be tested.